pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -306,6 +306,11 @@ const (
 	// multiple availability zones if it is possible.
 	// Deprecated: This annotation value is deprecated and not respected anymore. Please use `shoot.spec.controlPlane.highAvailability.failureTolerance.type=zone` instead.
 	ShootAlphaControlPlaneHighAvailabilityMultiZone = "multi-zone"
+	// ShootAlphaControlPlaneHAVPN is a constant for an annotation on the Shoot resource stating that the
+	// high availability setup for the VPN connection should be enabled.
```

Comment:
The document says that all HA shoots get the HA VPN as well. This doc string, however, suggests that this annotation must be present for it to get enabled. Can you harmonize this?

Reply:
new comment:
```
	// ShootAlphaControlPlaneHAVPN is a constant for an annotation on the Shoot resource to enforce
	// enabling/disabling the high availability setup for the VPN connection.
	// By default, the HA setup for VPN connections is activated automatically if the control plane high availability is enabled.
```